lib/stdlib/test/binary_module_SUITE.erl

```diff
@@ -1460,7 +1460,10 @@ error_info(_Config) ->
 
          {split, [<<1:15>>, <<"a">>, []]},
          {split, [<<1,2,3>>, {bm,make_ref()}, []]},
-         {split, [<<1,2,3>>, <<"2">>, [bad_option]]}
+         {split, [<<1,2,3>>, <<"2">>, [bad_option]]},
+
+         {encode_hex, [{no,a,binary}]},
+         {decode_hex, [{no,a,binary}]}
```

Comment:
There should also be a test for when it is a binary but not valid hex.

Reply:
After more than one hour to try, I'm unable to add this test case with the `error_info` style. Do you have documentation about this?

Reply:
Indeed! Having documentation about this would be helpful.
The change you do is fine for me.

Maybe it's time to squash commits?

Reply:
The extended error information is still in its infancy, so a lot of the details are not as good as they could be yet.

Yes, please squash everything.

Reply:
Done.